packages/node/__tests__/index.test.ts

```diff
@@ -108,18 +108,6 @@ describe('#metrics', () => {
     rimraf.sync(cacheDir);
   });
 
-  /* it('should error if missing apiKey', () => {
-    expect(() => {
-      expressMiddleware();
-    }).toThrow('You must provide your ReadMe API key');
-  });
-
-  it('should error if missing grouping function', () => {
-    expect(() => {
-      expressMiddleware('api-key');
-    }).toThrow('You must provide a grouping function');
-  }); */
```

Comment:
These tests no longer needed?

Reply:
They weren't needed to begin with, I commented them out thinking maybe I'll come back and fix them. Typescript protects against them and there's not a great way of testing this for non-typescript users.